CONTRIBUTING.md

```diff
@@ -2,9 +2,12 @@
 All non security bugs should be filed on the Issues tracker:
 https://github.com/01org/tpm2-abrmd/issues
 
-Security sensitive bugs should be emailed to a maintainer directly, or to Intel
-via the guidelines here:
-https://security-center.intel.com/VulnerabilityHandlingGuidelines.aspx
+Security sensitive bugs or questions should be emailed to *all* maintainers
+listed in the [MAINTAINERS](MAINTAINERS) file directly so consensus on the
+issue and the appropriate resolution can be reached. Alternatively you may
+contact Intel by following the instructions
+[here](https://security-center.intel.com/VulnerabilityHandlingGuidelines.aspx)
```

Comment:
This is no longer an Intel project, so we should require users to email security sensitive bugs to Maintainers or private issues, but I don't think Github supports that (Gitlab feature AFAICT). However, their might be information in here but I have not delved into it very deeply. https://docs.github.com/en/code-security/repository-security-advisories/about-coordinated-disclosure-of-security-vulnerabilities

Reply:
I think we can use this one about using github for security disclosure and coordination: https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories

Reply:
Though this PR is related, the change you mention probably deserves its own. Do you have any specific wording in mind? Any objections to merging this as-is and turning your comments here into a PR?